### PR TITLE
give media previews a rounded outline

### PR DESCRIPTION
**Pull request by connyduck**
Part of https://github.com/tuskyapp/Tusky/issues/1494

Screenshot_1578076261

Also made sure the sensitive media warning is the same in all layouts

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 1]
```diff
@@ -112,6 +112,8 @@ protected StatusBaseViewHolder(View itemView) {
         bookmarkButton = itemView.findViewById(R.id.status_bookmark);
         moreButton = itemView.findViewById(R.id.status_more);
 
+        itemView.findViewById(R.id.status_media_preview_container).setClipToOutline(true);
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: there is no xml attribute for this 🙄

File: app/src/main/res/layout/item_conversation.xml [hunk 2]
```diff
@@ -314,11 +315,14 @@
             android:id="@+id/status_sensitive_media_warning"
             android:layout_width="0dp"
             android:layout_height="0dp"
-            android:background="?attr/sensitive_media_warning_background_color"
+            android:background="@drawable/media_warning_bg"
             android:gravity="center"
             android:lineSpacingMultiplier="1.2"
             android:orientation="vertical"
-            android:padding="8dp"
+            android:paddingLeft="12dp"
```
Comment (charlag) [inline app/src/main/res/layout/item_conversation.xml]: Why new padding?

**Comment by charlag** [comment]
Looks cool!

File: app/src/main/res/layout/item_conversation.xml (hunk 2 above)
Comment (connyduck) [reply app/src/main/res/layout/item_conversation.xml]: so its the same as in `item_status`

**Comment by connyduck** [comment]
Yes, I always keep forgetting about the report 😐

**Pull request merged**